robot_skills/src/robot_skills/arm/arms.py

```diff
@@ -186,22 +174,6 @@ def move_down_until_force_sensor_edge_up(self, timeout=10, retract_distance=0.01
                        "Specify get_arm(..., force_sensor_required=True)")
         return self._arm.move_down_until_force_sensor_edge_up(timeout=timeout, retract_distance=retract_distance)
 
-    def handover_to_human(self, timeout=10, gripper_type=None):
-        if gripper_type is None:
-            gripper_type = self.default_gripper_type
-
-        self._test_die(gripper_type in self._available_gripper_types, 'gripper type ' + str(gripper_type),
```

Comment:
I think it would be valuable if this test behavior would be salvaged, because now it is thrown out.

Reply:
agreed, will change the gripper and handover properties to include such a check.